dbt/adapters/spark/connections.py

```diff
@@ -321,7 +321,7 @@ def open(cls, connection):
                                                'cluster', 'organization'])
 
                     conn_url = cls.SPARK_CONNECTION_URL.format(
-                        host=creds.host,
+                        host=creds.host if creds.host.startswith('https://') else 'https://' + creds.host,
```

Comment:
It looks like flake8 is giving you a hard time for this line being too long:
```
dbt/adapters/spark/connections.py:324:80: E501 line too long (106 > 79 characters)
```

Reply:
Dang! Thanks for pointing it out. Just pushed a fix